src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -1115,11 +1115,14 @@ public KojiBuildInfo getBuildInfo( int buildId, KojiSessionInfo session )
     private KojiBuildInfo withGAV( KojiBuildInfo buildInfo, KojiSessionInfo session )
             throws KojiClientException
     {
+        if (true) return buildInfo;
```

Comment:
@janinko This field is unstructured in Koji, so I think the best we can do is provide a Map<String, Object> which can contain any analog of a valid XML-RPC type, including nested Map<String,Object> instances.

If there are things specific to Maven or something like that we might be able to add an extractor utility class to help make sense of the extras field...